src/org/omegat/filters2/master/PluginUtils.java

```diff
@@ -269,8 +292,12 @@ protected static void loadFromManifest(final Manifest m, final ClassLoader class
                     continue;
                 }
                 if (loadClass(clazz, classLoader)) {
-                    PLUGIN_INFORMATIONS.add(new PluginInformation(clazz, m));
-                }
+                    if (mu == null) {
+                        PLUGIN_INFORMATIONS.add(new PluginInformation(clazz, m, null, PluginInformation.STATUS.BUNDLED));
```

Comment:
When `bundled` plugin is loaded, `mu` become null, because L.137 call here.
If there is better way to determine bundled plugins, it can be improved.